packages/bpk-component-checkbox/src/BpkCheckbox.scss

```diff
@@ -68,7 +68,7 @@
 
   &__icon {
     position: absolute;
-    top: ($bpk-spacing-base - $bpk-spacing-sm) / 2;
+    top: $bpk-spacing-xs;
```

Comment:
I feel like the old value was better because it captured more intent even though it was more complex. From it it was clear that this an attempt to center, now it's just an "arbitrary" value

Reply:
Using this old value meant that the alignment was off and the box looked all funky this now brings it back to align with the text and the box itself

Reply:
I can add a comment here to explain what this variable is actually doing

Reply:
It shouldn't the math works out to the same value